plasmapy/formulary/parameters.py

```diff
@@ -412,6 +412,16 @@ def ion_sound_speed(
     return V_S
 
 
+# This dictionary defines coefficients for thermal speeds
+# calculated for different methods and values of ndim.
+# Created here to avoid re-instantiating on each call
+coefficients = {
```

Comment:
I think I'd like to make these a private variable:
```suggestion
_coefficients = {
```

The underscore is sort of a convention saying "you probably shouldn't need to use this variable outside", since Python lacks `private`s.